DuckDuckGo/AppConfigurationFetch.swift

```diff
@@ -247,6 +269,7 @@ class AppConfigurationFetch {
                           Keys.fgFetchWithData: String(store.foregroundNewDataCount),
                           Keys.bgFetchType: backgroundFetchType,
                           Keys.bgFetchTaskExpiration: String(Self.backgroundFetchTaskExpirationCount),
+                          Keys.bgFetchTaskDuration: String(Self.backgroundFetchTaskDuration),
```

Comment:
I'm only tracking duration for background calls as I want to be mindful of collecting stats that we aren't explicitly interested in using right now.

This value is an aggregate of all background refreshes that are run between `sendStatistics` calls – we can get an average duration for each pixel by calculating `total duration / number of background refresh calls`. To me that seems good enough, but LMK if something's off with this or there's a better approach.